backend/Watchdog.Core/Watchdog.Core.API/Extensions/ServiceCollectionExtensions.cs

```diff
@@ -22,9 +22,8 @@ public static void RegisterCustomServices(this IServiceCollection services, ICon
                 .AddControllers()
                 .AddNewtonsoftJson(options => options.SerializerSettings.ReferenceLoopHandling = Newtonsoft.Json.ReferenceLoopHandling.Ignore);
 
-            services.AddTransient<IApplicationService, ApplicationService>();
```

Comment:
Why do you delete ApplicationService?

Reply:
There were dublicates 
![image](https://user-images.githubusercontent.com/75535425/129485657-f18ca9fd-4b2f-4d53-b6d5-54fc05ad4ad7.png)